graylog2-web-interface/src/components/search/MessageFields.jsx

```diff
@@ -46,7 +46,9 @@ class MessageFields extends React.Component {
 
   render() {
     const { message } = this.props;
-    const formattedFields = message.formatted_fields;
+    // eslint-disable-next-line no-unused-vars
+    const { _id, ...formatted_fields } = message.fields;
```

Comment:
Why do we need the `_id`? I think we can use `_` and eslint might not complain, but not sure.